<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>619</width>
         <height>462</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Municipality/Rural Municipality</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ward No.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kitta No.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Area</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>House Details</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description of the route</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Remarks</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="permanentAddressLabel">
            <property name="text">
             <string>PermanentAddress</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="permanentAddressLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="addressOfMalpotLabel">
            <property name="text">
             <string>Address of Malpot Office</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="addressOfMalpotLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="citizenshipNoLabel">
            <property name="text">
             <string>Citizenship No.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="citizenshipNoLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_submit">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
